test/library/helpers.go

```
@@ -0,0 +1,320 @@
+		label = "node-role.kubernetes.io/master="
+	}
+	out, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("node", "-l", label, "-o=jsonpath={.items[*].metadata.name}").Output()
+	if err != nil {
+		g.Fail(fmt.Sprintf("get cluster nodes by %s: %v", role, err))
+	}
+	return strings.Fields(out)
+}
+
+func DebugNodeWithChroot(oc *OC, node string, args ...string) (string, error) {
+	debugArgs := append([]string{"debug", "node/" + node, "--", "chroot", "/host"}, args...)
+	cmd := exec.Command("oc", debugArgs...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)